crates/autopilot/src/periodic_db_cleanup.rs

```diff
@@ -94,18 +94,23 @@ mod tests {
             timestamp: now - chrono::Duration::milliseconds(300),
             label: OrderEventLabel::Created,
         };
+        database::order_events::insert_order_event(&mut ex, &event_a)
```

Comment:
@MartinquaXD taught me that tests without postgres prefix are not executed by CI. Was this intended?

..this function name is "order_events_cleaner_flow"

Reply:
Nice catch, thanks